frontend/packages/client/src/components/Proposal/VoteHeader.js

```diff
@@ -0,0 +1,28 @@
+import { Svg } from '@cast/shared-components';
+
+const Wrapper = ({ children }) => (
+  <h3
+    className={`is-size-5 has-text-weight-bold`}
+    style={{ lineHeight: '24px' }}
+  >
+    {children}
+  </h3>
+);
+
+export default function VoteHeader({ status }) {
+  // Status: user-voted, invite-to-vote, is-closed
+  const message = {
+    'user-voted': (
+      <div className="is-flex is-align-items-centered">
+        <div className="mr-2 is-flex is-align-items-center">
+          <Svg name="CheckMark" height="16" width="16" circleFill="#3EAE4F" />
+        </div>
+        You successfully voted on this proposal!
+      </div>
+    ),
+    'invite-to-vote': <>Rank your vote &#10024;</>,
+    'is-closed': <>Voting has ended on this proposal.</>,
+  };
+
+  return <Wrapper>{message[status] ?? <>Rank your vote</>}</Wrapper>;
+}
```

Comment:
I wonder if we can use only one of these components and use something like:

```
import { Svg } from '@cast/shared-components';

const Wrapper = ({ children }) => (
  <h3
    className={`is-size-5 has-text-weight-bold`}
    style={{ lineHeight: '24px' }}
  >
    {children}
  </h3>
);

export default function VoteHeader({ status, voteOption }) {

  const invite = voteOption === 'single-choice' ? <>Cast your vote</> : <>Rank your vote &#10024;</>
  // Status: user-voted, invite-to-vote, is-closed
  const message = {
    'user-voted': <>You successfully voted on this proposal!</>,
    'invite-to-vote': invite,
    'is-closed': <>Voting has ended on this proposal.</>,
  };

  return <Wrapper>{message[status] ?? invite}</Wrapper>;
}
```
then we can remove it from RankedChoiceVote and SingleChoiceVote folder

Reply:
Done.